.gitmodules

```diff
@@ -1,4 +1,3 @@
 [submodule "Archipelago"]
 	path = Archipelago
-	url = https://github.com/Ixrec/Archipelago.git
-	branch = nine_sols
```

Comment:
I'm pretty sure we need the branch here?

Reply:
No, git index keeps track of the commit. You can see that by going into the github diff where it says `Submodule [Archipelago](https://github.com/Ixrec/Archipelago) added at 08bc20`, which is the latest commit on the `nine_sols` branch
Adding it here is just cosmetic, though if you prefer, I can keep the line.

Reply:
fair enough

I never fully grokked submodules, and I don't even understand what issue this is fixing (it's been fine on my end), but I think the most productive option is to just merge this and then see if both of our local clones are happy afterward